src/shared/clients/codecatalystClient.ts

```diff
@@ -411,12 +413,45 @@ class CodeCatalystClientInternal {
 
     /**
      * Gets a flat list of all repos for the given CodeCatalyst user.
+     * @param internalOnly True if you want only internal repositories to be returned.
```

Comment:
```suggestion
     * @param internalOnly Get only internal (non-3P) repositories.
```

Reply:
Resolved but the change wasn't made :P

Reply:
I renamed from 'internal' to 'firstParty' since that seemed to be the right convention to use. But I updated the change to reflect your comment.